src/DynamicSizeList.js

```diff
@@ -88,9 +88,7 @@ const findNearestItemBinarySearch = (
     const middle = low + Math.floor((high - low) / 2);
     const currentOffset = getItemMetadata(props, middle, instanceProps).offset;
 
-    if (!currentOffset) {
```

Comment:
If there are more than 200 posts middle will greater than 101 and on load 101 is not mounted so !currentOffset is true making startIndex be `0` as low is `0` causing https://mattermost.atlassian.net/browse/MM-15257